__test__/test_helpers.js

```diff
@@ -1,36 +1,7 @@
 import { createLoaders } from '../src/server/models/';
-import thinky from 'thinky';
```

Comment:
most of this file was no-longer necessary because starting up the process automatically creates the schema.